### ISSUE TITLE
Option to disable crazypass in new wallets

### ISSUE THREAD

**Issue by mister-monster** [m2049r/xmrwallet#612]
Intuitively, people expect the wallet to function as much like the desktop wallet as possible. The passphrase they enter is the one they expect to have to enter when they restore.

An option to disable/enable crazypass on creation of a new wallet would be great, there is no good reason crazypass is forced on the user, diverging from every other XMR wallet. At least a few users have lost XMR due to this misunderstanding.

**Comment by SamsungGalaxyPlayer** [comment]
The correct answer here is more education imo. Crazypass is far more secure by default than a custom password. Do you feel the warning to back up the mnemonic seed and Crazypass are not sufficient?

**Comment by mister-monster** [comment]
No, I do not think it is sufficient for the average user, because users follow patterns and the pattern here is different from the desktop XMR wallet. A user writes down their seed, that has been the way to do it for a long time. Anyone who follows the usual pattern is going to find out the hard way that this app diverges from the standard. People ignore warnings all the time, and it is easy to say "well then don't ignore warnings" but that doesn't fix UX.

And you wouldn't even know anything is different here from using the wallet because the password you use daily is the one you picked. You back up your mnemonic, or you back up your .keys file and you think all is well, because that is how it usually works. And then you find out you've lost your keys because this specific wallet does things differently? A warning does not suffice, no.

At the very least, the option to have this wallet work the same as other XMR wallets should be there. If it is going to diverge from the standard it should be optional.

**Comment by m2049r** [comment]
> At least a few users have lost XMR due to this misunderstanding.

If a few users have lost XMR, it's due to not writing down their seed - nothing else.

If the average user you are talking about doesn't respect warnings, doesn't read help texts, does not backup their seed and disables crazypass, they will also use simple passwords which will be hacked in seconds and so they will loose XMR with absolute certainty.

Ontop of this, there is no way to check if a wallet was secured with crazypass or not - and you can be sure that hacked users will blame me for losing/stealing their XMR. Even if there is a big disclaimer saying - "Yo! Don't use this option or you will lose all your funds - with certainty!"

If you have an idea how to change the UI/UX while keeping the security level, we would be grateful.

**Comment by m2049r** [comment]
I also suggest trying out passwords on https://www.bennish.net/password-strength-checker/ to get a feeling for how unsecure the passwords you use really are. Take into account, that the monero passwords are not server passwords and cannot be throttled or invalidated - if you have the keys file, you can run it through a millions password hackers in parallel if you have the hardware...
